### ISSUE TITLE
"500 Internal Server Error (key required)" when sending report

### ISSUE THREAD

**Issue by guilhem-lk** [gen2brain/acra-go#1]
I receive an error when I send a report to the backend :
"500 Internal Server Error (key required)"

It seems that acra-go can't extract the REPORT_ID from the report.

Here is my report: [test-report.txt](https://github.com/gen2brain/acra-go/files/963762/test-report.txt)

I tested with one of your report and it's working fine. The only diff I saw with mine is that your fields are sorted in alphabetical order, not mine...

**Comment by gen2brain** [comment]
Are you using org.acra.sender.HttpSender.Type.JSON or plain form? It seems there are some issues with form library.

**Comment by guilhem-lk** [comment]
Yes, I send it as JSON in a POST method. The test-report.txt file is sent in the request body.

I also tested it directly with a curl command on the server: 
curl -X POST http://localhost:55000/send -d @test-report.txt -H"Content-Type: application/json"

Same result.

Thank you for your help.

**Comment by guilhem-lk** [comment]
FYI, I tested it on Debian with Go v1.0.2

I also checked on Windows with latest version (1.8.1) and I'm having the same issue.

**Comment by gen2brain** [comment]
I didn't had time to check, I guess report.go needs to be compared with actual output. I only tested with example output from acra project, not from actual output.

**Comment by gen2brain** [comment]
I implemented a workaround in 829590616b75a915619eb3ac3f7b14a143387a15 .
